RoofingCompany.sql

```diff
@@ -24,14 +24,14 @@ create table EntranceControl(
 	);
 
 create table FEMAnalysis(
-	IdFEMAnalysis int primary key not null,
+	IdFEMAnalysis int primary key identity (1,1) not null,
 	IdEmployee int not null,
 	NewPattern image not null,
 	AnalysisResults text not null
 	);
 
 create table OutControl(
-	IdProcess int primary key not null,
+	IdProces int primary key not null,
```

Comment:
1. jeśli ma być po angielsku to `Process` jest poprawnie
2. zmieniona struktura bazy danych-> brakuje zaktualizowanego modelu w EF